README.md

```diff
@@ -19,12 +19,14 @@
 </div>
 
 ## What is TotalCross?
-
-TotalCross is an open source and free GUI Creator for embedded systems and a framework for developing cross-platform applications for android, iOS, wince, windows and desktop, Linux desktop and Linux arm using Kotlin or Java or any Java Based language.
+TotalCross is tools to make easy and fast GUI creation:
+ 
+* KnowCode:  A computer vision project to help developers to build their screen faster based on image made by design using a library to allow run android XML UI + TotalCross SDK on Linux Arm, iOS, Android and more;
+* TotalCross SDK (or just TotalCross): Is an open source and free GUI Creator for embedded systems and a framework for developing cross-platform applications for android, iOS, wince, windows and desktop, Linux desktop and Linux arm using Kotlin or Java or any Java Based language.
```

Comment:
Please, change it to:
_* TotalCross SDK: Is a free open source GUI Creator and framework for developing cross-platform applications for Embedded Systems, Android, iOS, Wince, Windows and Linux desktop applications, and Linux Arm. Current supported languages are Kotlin, Java and any Java Based language._

Reply:
done